manifests/candlepin.pp

```diff
@@ -89,22 +71,6 @@
       key_decrypt       => true,
     }
 
-    file { "${pki_dir}/private/katello-tomcat.key":
-      ensure  => absent,
```

Comment:
Ensured absent in 8e0050580196ae4f9132ff395885f481314b52d and, released in 14.0.0, but I think prior to that already.